packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -195,6 +195,12 @@ export class Cluster extends Resource implements ICluster {
       autoScalingGroup.addUserData('echo ECS_AWSVPC_BLOCK_IMDS=true >> /etc/ecs/ecs.config');
     }
 
+    if (options.spotInstanceDraining) {
+      // Automated Spot Instance Draining
+      // Source: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-instance-spot.html
```

Comment:
Since it has been specified clearly in the property comment, this comment may not be necessary.

Reply:
fixed https://github.com/aws/aws-cdk/pull/4360/commits/45bef1da7735bc55ff50668e70c7c3c9349b6944